lfs/client.go

```diff
@@ -126,6 +127,17 @@ func Download(oid string) (io.ReadCloser, int64, error) {
 	}
 	LogTransfer("lfs.data.download", res)
 
+	if Config.NtlmAccess() {
+		buf, _ := ioutil.ReadAll(res.Body)
```

Comment:
If we read the entire response into memory, large files can definitely crash the client on low memory machines. We could _maybe_ do this _if_ content length is set _and_ it's under some value - but I would really prefer to not do it at all for downloads. This is also ignoring the error.